recommendation/system.web.mvc.json

```diff
@@ -269,14 +303,427 @@
+              "Type": "Using",
+              "Value": "System.Web.Mvc",
+              "Description": "Remove System.Web.Mvc namespace",
+              "ActionValidation": {
+                "Contains": "",
+                "NotContains": "using System.Web.Mvc;"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "Type": "Method",
+      "Name": "System.Web.Mvc.DependencyResolver.SetResolver(System.Web.Mvc.IDependencyResolver)",
+      "Value": "System.Web.Mvc.DependencyResolver.SetResolver(System.Web.Mvc.IDependencyResolver)",
+      "KeyType": "Name",
+      "ContainingType": "DependencyResolver",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
+          ],
+          "Description": "Add a comment: Use the startup class to register dependency containers.",
+          "Actions": [
+            {
+              "Name": "AddComment",
+              "Type": "Method",
+              "Value": "This method is not supported. Use the startup class to register dependency containers",
+              "Description": "Use the startup class to register dependency containers",
+              "ActionValidation": {
+                "Contains": "This method is not supported. Use the startup class to register dependency containers",
+                "NotContains": "",
+                "CheckComments": "True"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "Type": "Method",
+      "Name": "System.Web.Mvc.Controller.TryUpdateModel(*)",
+      "Value": "System.Web.Mvc.Controller.TryUpdateModel(*)",
+      "KeyType": "Name",
+      "ContainingType": "Controller",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
+          ],
+          "Description": "Replace TryUpdateModel with TryUpdateModelAsync and replace System.Web.Mvc namespace with Microsoft.AspNetCore.Mvc.",
+          "Actions": [
+            {
+              "Name": "ReplaceMethodOnly",
+              "Type": "Method",
+              "Value": {
+                "oldMethod": "TryUpdateModel",
+                "newMethod": "TryUpdateModelAsync"
+              },
+              "Description": "Replace TryUpdateModel with TryUpdateModelAsync",
+              "ActionValidation": {
+                "Contains": "TryUpdateModelAsync"
```

Comment:
Same here for consistency should we add a notcontains?

Reply:
Sure, updated.